src/Riok.Mapperly/AnalyzerReleases.Shipped.md

```diff
@@ -116,3 +116,12 @@ Rule ID | Category | Severity | Notes
 RMG048  | Mapper   | Error    | Used mapper members cannot be nullable
 RMG049  | Mapper   | Warning  | Source member is ignored and also explicitly mapped
 RMG050  | Mapper   | Warning  | Target member is ignored and also explicitly mapped
+
+## Release 3.3
+
+### New Rules
+
+| Rule ID | Category | Severity | Notes                                                                |
```

Comment:
You need to use the exact same header as the other releases otherwise the analyzer won't recognize it correctly.

Reply:
Thanks, not the first time VS code's formatter has "helped" out 😤